tests/phpunit/WC_Stripe_Order_Helper_Test.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace WooCommerce\Stripe\Tests;
+
+use WC_Order;
+use WC_Stripe_Exception;
+use WP_UnitTestCase;
+use WC_Data_Exception;
+use WC_Stripe_Order_Helper;
+use WooCommerce\Stripe\Tests\Helpers\WC_Helper_Order;
+
+/**
+ * Class WC_Stripe_Order_Helper
+ *
+ * @package WooCommerce/Stripe/WC_Stripe_Order_Helper
+ *
+ * Class WC_Stripe_Order_Helper tests.
+ */
+class WC_Stripe_Order_Helper_Test extends WP_UnitTestCase {
+	/**
+	 * Tests for `get_order_by_source_id`, `get_order_by_charge_id`, `get_order_by_refund_id`,
+	 * `get_order_by_intent_id`, and `get_order_by_setup_intent_id`.
```

Comment:
These methods are not present in `WC_Stripe_Order_Helper` class.

Reply:
Oops, forgot to update the test class. Did it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4613/commits/357fa894b962a91d097307a91e25b7dff9da0617